crates/sui-core/src/checkpoints/mod.rs

```diff
@@ -118,7 +118,7 @@ pub struct CheckpointStore {
     /// to allow us to provide a list in order they were received. We only store
     /// the fragments that are relevant to the next checkpoints. Past checkpoints
     /// already contain all relevant information from previous checkpoints.
-    pub fragments: DBMap<u64, CheckpointFragment>,
+    pub fragments: DBMap<ExecutionIndices, CheckpointFragment>,
```

Comment:
Ok, now it seems we are storing fragments both in the narwhal DB, then in state DB, then in checkpoint DB.

Reply:
I don't think we are storing them in Narwhal DB. Here I just changed the type of a sequence number from u64 to ExecutionIndex